packages/contracts-bedrock/test/L1/SystemConfig.t.sol

```diff
@@ -209,6 +209,95 @@ contract SystemConfig_Initialize_TestFail is SystemConfig_Initialize_Test {
     }
 }
 
+contract SystemConfig_Init_ResourceConfig is SystemConfig_Init {
+    /// @dev Tests that `setResourceConfig` reverts if the min base fee
+    ///      is greater than the maximum allowed base fee.
+    function test_setResourceConfig_badMinMax_reverts() external {
+        ResourceMetering.ResourceConfig memory config = ResourceMetering.ResourceConfig({
+            maxResourceLimit: 20_000_000,
+            elasticityMultiplier: 10,
+            baseFeeMaxChangeDenominator: 8,
+            systemTxMaxGas: 1_000_000,
+            minimumBaseFee: 2 gwei,
+            maximumBaseFee: 1 gwei
+        });
```

Comment:
IMO this suite would be easier to read of we passed in the [`DEFAULT_RESOURCE_CONFIG` ](https://github.com/ethereum-optimism/optimism/blob/developer/packages/contracts-bedrock/src/libraries/Constants.sol#L42) values everywhere except for the ones that we're manipulating for testing purposes.

Reply:
This is the config that the previous tests for the resource config setter were made for. Preferred not modifying them.

Reply:
Disagree, but won't block.